packages/core/utils/src/modules-sdk/mikro-orm-cli-config-builder.ts

```diff
@@ -53,17 +51,11 @@ export function defineMikroOrmCliConfig(
   ) as MikroORMOptions["entities"]
 
   const normalizedModuleName = kebabCase(moduleName.replace("Service", ""))
-  let databaseName = `medusa-${normalizedModuleName}`
-
-  if (options.databaseName) {
-    databaseName = options.databaseName
-    // @ts-ignore
-    delete options.databaseName
-  }
+  const databaseName = `medusa-${normalizedModuleName}`
 
   return {
-    clientUrl: `postgres://postgres@localhost/${databaseName}`,
     type: "postgresql",
+    dbName: databaseName,
```

Comment:
q: does it build a default client url from just the dbname by default?

Reply:
No idea. I can check it now.

Reply:
Seems to work fine when no explicit `clientUrl` is provided